css.md

```diff
@@ -0,0 +1,32 @@
+# CSS
+
+**Prerequisite knowledge**: basic html elements
```

Comment:
Quoting https://github.com/bethinkpl/bit-docs/blob/master/!document_template.md#all-custom-stuff:
>In order to be able to connect topics in an automated manner, items listed in "Prerequisite knowledge" and "Next steps" sections must match exact filenames of linked documents.

So this should be:
```suggestion
**Prerequisite knowledge**: html
```